[roslaunch][INFO] 2024-03-05 04:55:17,266: Checking log directory for disk usage. This may take a while.
Press Ctrl-C to interrupt
[roslaunch][INFO] 2024-03-05 04:55:17,267: Done checking log file disk usage. Usage is <1GB.
[roslaunch][INFO] 2024-03-05 04:55:17,268: roslaunch starting with args ['/opt/ros/noetic/bin/roslaunch', 'smb_gazebo', 'smb_gazebo.launch']
[roslaunch][INFO] 2024-03-05 04:55:17,268: roslaunch env is environ({'SHELL': '/bin/bash', 'ROS_VERSION': '1', 'SESSION_MANAGER': 'local/TheanVIn:@/tmp/.ICE-unix/1959,unix/TheanVIn:/tmp/.ICE-unix/1959', 'QT_ACCESSIBILITY': '1', 'COLORTERM': 'truecolor', 'XDG_CONFIG_DIRS': '/etc/xdg/xdg-ubuntu:/etc/xdg', 'XDG_MENU_PREFIX': 'gnome-', 'GNOME_DESKTOP_SESSION_ID': 'this-is-deprecated', 'PKG_CONFIG_PATH': '/home/theanvin/Workspaces/smb_ws/devel/lib/pkgconfig:/home/theanvin/catkin_ws/devel/lib/pkgconfig:/opt/ros/noetic/lib/pkgconfig', 'ROS_PYTHON_VERSION': '3', 'GNOME_SHELL_SESSION_MODE': 'ubuntu', 'SSH_AUTH_SOCK': '/run/user/1000/keyring/ssh', 'XMODIFIERS': '@im=ibus', 'DESKTOP_SESSION': 'ubuntu', 'SSH_AGENT_PID': '1892', 'ROS_PACKAGE_PATH': '/home/theanvin/Workspaces/smb_ws/src:/home/theanvin/catkin_ws/src:/opt/ros/noetic/share', 'GTK_MODULES': 'gail:atk-bridge', 'ROSLISP_PACKAGE_DIRECTORIES': '/home/theanvin/Workspaces/smb_ws/devel/share/common-lisp:/home/theanvin/catkin_ws/devel/share/common-lisp', 'PWD': '/home/theanvin/Workspaces/smb_ws', 'LOGNAME': 'theanvin', 'XDG_SESSION_DESKTOP': 'ubuntu', 'XDG_SESSION_TYPE': 'x11', 'GPG_AGENT_INFO': '/run/user/1000/gnupg/S.gpg-agent:0:1', 'XAUTHORITY': '/run/user/1000/gdm/Xauthority', 'GJS_DEBUG_TOPICS': 'JS ERROR;JS LOG', 'WINDOWPATH': '2', 'HOME': '/home/theanvin', 'USERNAME': 'theanvin', 'IM_CONFIG_PHASE': '1', 'LANG': 'en_US.UTF-8', 'ROS_ETC_DIR': '/opt/ros/noetic/etc/ros', 'LS_COLORS': 'rs=0:di=01;34:ln=01;36:mh=00:pi=40;33:so=01;35:do=01;35:bd=40;33;01:cd=40;33;01:or=40;31;01:mi=00:su=37;41:sg=30;43:ca=30;41:tw=30;42:ow=34;42:st=37;44:ex=01;32:*.tar=01;31:*.tgz=01;31:*.arc=01;31:*.arj=01;31:*.taz=01;31:*.lha=01;31:*.lz4=01;31:*.lzh=01;31:*.lzma=01;31:*.tlz=01;31:*.txz=01;31:*.tzo=01;31:*.t7z=01;31:*.zip=01;31:*.z=01;31:*.dz=01;31:*.gz=01;31:*.lrz=01;31:*.lz=01;31:*.lzo=01;31:*.xz=01;31:*.zst=01;31:*.tzst=01;31:*.bz2=01;31:*.bz=01;31:*.tbz=01;31:*.tbz2=01;31:*.tz=01;31:*.deb=01;31:*.rpm=01;31:*.jar=01;31:*.war=01;31:*.ear=01;31:*.sar=01;31:*.rar=01;31:*.alz=01;31:*.ace=01;31:*.zoo=01;31:*.cpio=01;31:*.7z=01;31:*.rz=01;31:*.cab=01;31:*.wim=01;31:*.swm=01;31:*.dwm=01;31:*.esd=01;31:*.jpg=01;35:*.jpeg=01;35:*.mjpg=01;35:*.mjpeg=01;35:*.gif=01;35:*.bmp=01;35:*.pbm=01;35:*.pgm=01;35:*.ppm=01;35:*.tga=01;35:*.xbm=01;35:*.xpm=01;35:*.tif=01;35:*.tiff=01;35:*.png=01;35:*.svg=01;35:*.svgz=01;35:*.mng=01;35:*.pcx=01;35:*.mov=01;35:*.mpg=01;35:*.mpeg=01;35:*.m2v=01;35:*.mkv=01;35:*.webm=01;35:*.ogm=01;35:*.mp4=01;35:*.m4v=01;35:*.mp4v=01;35:*.vob=01;35:*.qt=01;35:*.nuv=01;35:*.wmv=01;35:*.asf=01;35:*.rm=01;35:*.rmvb=01;35:*.flc=01;35:*.avi=01;35:*.fli=01;35:*.flv=01;35:*.gl=01;35:*.dl=01;35:*.xcf=01;35:*.xwd=01;35:*.yuv=01;35:*.cgm=01;35:*.emf=01;35:*.ogv=01;35:*.ogx=01;35:*.aac=00;36:*.au=00;36:*.flac=00;36:*.m4a=00;36:*.mid=00;36:*.midi=00;36:*.mka=00;36:*.mp3=00;36:*.mpc=00;36:*.ogg=00;36:*.ra=00;36:*.wav=00;36:*.oga=00;36:*.opus=00;36:*.spx=00;36:*.xspf=00;36:', 'XDG_CURRENT_DESKTOP': 'ubuntu:GNOME', 'VTE_VERSION': '6003', 'CMAKE_PREFIX_PATH': '/home/theanvin/Workspaces/smb_ws/devel:/home/theanvin/catkin_ws/devel:/opt/ros/noetic', 'GNOME_TERMINAL_SCREEN': '/org/gnome/Terminal/screen/1c7af9e3_f121_4653_bc41_71c802ff6fd8', 'INVOCATION_ID': '0466cdb5ec2d4bbeb1028c570b952c7b', 'MANAGERPID': '1652', 'GJS_DEBUG_OUTPUT': 'stderr', 'LESSCLOSE': '/usr/bin/lesspipe %s %s', 'XDG_SESSION_CLASS': 'user', 'PYTHONPATH': '/opt/ros/noetic/lib/python3/dist-packages', 'TERM': 'xterm-256color', 'LESSOPEN': '| /usr/bin/lesspipe %s', 'USER': 'theanvin', 'GNOME_TERMINAL_SERVICE': ':1.128', 'DISPLAY': ':1', 'SHLVL': '1', 'ROS_MASTER_URI': 'http://localhost:11311', 'QT_IM_MODULE': 'ibus', 'ROS_HOSTNAME': 'localhost', 'LD_LIBRARY_PATH': '/home/theanvin/Workspaces/smb_ws/devel/lib:/home/theanvin/catkin_ws/devel/lib:/opt/ros/noetic/lib', 'XDG_RUNTIME_DIR': '/run/user/1000', 'JOURNAL_STREAM': '8:50546', 'XDG_DATA_DIRS': '/usr/share/ubuntu:/usr/local/share/:/usr/share/:/var/lib/snapd/desktop', 'PATH': '/opt/ros/noetic/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin', 'GDMSESSION': 'ubuntu', 'DBUS_SESSION_BUS_ADDRESS': 'unix:path=/run/user/1000/bus', 'ROS_ROOT': '/opt/ros/noetic/share/ros', 'ROS_DISTRO': 'noetic', 'OLDPWD': '/home/theanvin/git/smb_gazebo', '_': '/opt/ros/noetic/bin/roslaunch', 'ROS_LOG_FILENAME': '/home/theanvin/.ros/log/785f2c00-dad6-11ee-82d1-f1f8f60cd418/roslaunch-TheanVIn-12988.log'})
[roslaunch][INFO] 2024-03-05 04:55:17,268: starting in server mode
[roslaunch.parent][INFO] 2024-03-05 04:55:17,268: starting roslaunch parent run
[roslaunch][INFO] 2024-03-05 04:55:17,268: loading roscore config file /opt/ros/noetic/etc/ros/roscore.xml
[roslaunch][INFO] 2024-03-05 04:55:17,370: Added core node of type [rosout/rosout] in namespace [/]
[roslaunch.config][INFO] 2024-03-05 04:55:17,370: loading config file /home/theanvin/Workspaces/smb_ws/src/smb_gazebo/launch/smb_gazebo.launch
[roslaunch][INFO] 2024-03-05 04:55:17,395: Added node of type [gazebo_ros/gzserver] in namespace [/]
[roslaunch][INFO] 2024-03-05 04:55:17,395: Added node of type [gazebo_ros/gzclient] in namespace [/]
[roslaunch][INFO] 2024-03-05 04:55:17,396: Added node of type [gazebo_ros/spawn_model] in namespace [/]
[roslaunch][INFO] 2024-03-05 04:55:17,398: Added node of type [pointcloud_to_laserscan/pointcloud_to_laserscan_node] in namespace [/]
[roslaunch][ERROR] 2024-03-05 04:55:17,592: RLException: while processing /home/theanvin/Workspaces/smb_ws/src/smb_control/launch/control.launch:
while processing /home/theanvin/Workspaces/smb_ws/src/smb_description/launch/load.launch:
Invalid <param> tag: Cannot load command parameter [robot_description]: command [['/opt/ros/noetic/lib/xacro/xacro', '/home/theanvin/Workspaces/smb_ws/src/smb_description/urdf/smb_standalone.urdf.xacro', 'wheel_joint_type:=continuous', 'simulation:=true', 'robot_namespace:=', 'lidar:=true', 'description_name_xacro:=robot_description', 'publish_tf:=false']] returned with code [2]. 

Param xml is <param name="$(arg description_name)" command="$(find xacro)/xacro     $(arg description_file)     wheel_joint_type:=$(arg wheel_joint_type)     simulation:=$(arg simulation)     robot_namespace:=$(arg robot_namespace)     lidar:=$(arg lidar)     description_name_xacro:=$(arg description_name)     publish_tf:=$(arg publish_tf)"/>
[roslaunch][ERROR] 2024-03-05 04:55:17,592: The traceback for the exception was written to the log file
[roslaunch][ERROR] 2024-03-05 04:55:17,593: Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3/dist-packages/roslaunch/xmlloader.py", line 271, in _param_tag
    value = self.param_value(verbose, name, ptype, *vals)
  File "/opt/ros/noetic/lib/python3/dist-packages/roslaunch/loader.py", line 543, in param_value
    raise ValueError("Cannot load command parameter [%s]: command [%s] returned with code [%s]"%(name, command, p.returncode))
ValueError: Cannot load command parameter [robot_description]: command [['/opt/ros/noetic/lib/xacro/xacro', '/home/theanvin/Workspaces/smb_ws/src/smb_description/urdf/smb_standalone.urdf.xacro', 'wheel_joint_type:=continuous', 'simulation:=true', 'robot_namespace:=', 'lidar:=true', 'description_name_xacro:=robot_description', 'publish_tf:=false']] returned with code [2]

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3/dist-packages/roslaunch/xmlloader.py", line 635, in _include_tag
    self._recurse_load(ros_config, launch.childNodes, child_ns, \
  File "/opt/ros/noetic/lib/python3/dist-packages/roslaunch/xmlloader.py", line 681, in _recurse_load
    self._param_tag(tag, context, ros_config, verbose=verbose)
  File "/opt/ros/noetic/lib/python3/dist-packages/roslaunch/xmlloader.py", line 96, in call
    return f(*args, **kwds)
  File "/opt/ros/noetic/lib/python3/dist-packages/roslaunch/xmlloader.py", line 285, in _param_tag
    raise XmlParseException(
roslaunch.xmlloader.XmlParseException: Invalid <param> tag: Cannot load command parameter [robot_description]: command [['/opt/ros/noetic/lib/xacro/xacro', '/home/theanvin/Workspaces/smb_ws/src/smb_description/urdf/smb_standalone.urdf.xacro', 'wheel_joint_type:=continuous', 'simulation:=true', 'robot_namespace:=', 'lidar:=true', 'description_name_xacro:=robot_description', 'publish_tf:=false']] returned with code [2]. 

Param xml is <param name="$(arg description_name)" command="$(find xacro)/xacro     $(arg description_file)     wheel_joint_type:=$(arg wheel_joint_type)     simulation:=$(arg simulation)     robot_namespace:=$(arg robot_namespace)     lidar:=$(arg lidar)     description_name_xacro:=$(arg description_name)     publish_tf:=$(arg publish_tf)"/>

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3/dist-packages/roslaunch/xmlloader.py", line 635, in _include_tag
    self._recurse_load(ros_config, launch.childNodes, child_ns, \
  File "/opt/ros/noetic/lib/python3/dist-packages/roslaunch/xmlloader.py", line 701, in _recurse_load
    val = self._include_tag(tag, context, ros_config, default_machine, is_core, verbose)
  File "/opt/ros/noetic/lib/python3/dist-packages/roslaunch/xmlloader.py", line 96, in call
    return f(*args, **kwds)
  File "/opt/ros/noetic/lib/python3/dist-packages/roslaunch/xmlloader.py", line 646, in _include_tag
    raise XmlParseException("while processing %s:\n%s"%(inc_filename, str(e)))
roslaunch.xmlloader.XmlParseException: while processing /home/theanvin/Workspaces/smb_ws/src/smb_description/launch/load.launch:
Invalid <param> tag: Cannot load command parameter [robot_description]: command [['/opt/ros/noetic/lib/xacro/xacro', '/home/theanvin/Workspaces/smb_ws/src/smb_description/urdf/smb_standalone.urdf.xacro', 'wheel_joint_type:=continuous', 'simulation:=true', 'robot_namespace:=', 'lidar:=true', 'description_name_xacro:=robot_description', 'publish_tf:=false']] returned with code [2]. 

Param xml is <param name="$(arg description_name)" command="$(find xacro)/xacro     $(arg description_file)     wheel_joint_type:=$(arg wheel_joint_type)     simulation:=$(arg simulation)     robot_namespace:=$(arg robot_namespace)     lidar:=$(arg lidar)     description_name_xacro:=$(arg description_name)     publish_tf:=$(arg publish_tf)"/>

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3/dist-packages/roslaunch/config.py", line 461, in load_config_default
    loader.load(f, config, argv=args, verbose=verbose)
  File "/opt/ros/noetic/lib/python3/dist-packages/roslaunch/xmlloader.py", line 763, in load
    self._load_launch(launch, ros_config, is_core=core, filename=filename, argv=argv, verbose=verbose)
  File "/opt/ros/noetic/lib/python3/dist-packages/roslaunch/xmlloader.py", line 735, in _load_launch
    self._recurse_load(ros_config, launch.childNodes, self.root_context, None, is_core, verbose)
  File "/opt/ros/noetic/lib/python3/dist-packages/roslaunch/xmlloader.py", line 701, in _recurse_load
    val = self._include_tag(tag, context, ros_config, default_machine, is_core, verbose)
  File "/opt/ros/noetic/lib/python3/dist-packages/roslaunch/xmlloader.py", line 96, in call
    return f(*args, **kwds)
  File "/opt/ros/noetic/lib/python3/dist-packages/roslaunch/xmlloader.py", line 646, in _include_tag
    raise XmlParseException("while processing %s:\n%s"%(inc_filename, str(e)))
roslaunch.xmlloader.XmlParseException: while processing /home/theanvin/Workspaces/smb_ws/src/smb_control/launch/control.launch:
while processing /home/theanvin/Workspaces/smb_ws/src/smb_description/launch/load.launch:
Invalid <param> tag: Cannot load command parameter [robot_description]: command [['/opt/ros/noetic/lib/xacro/xacro', '/home/theanvin/Workspaces/smb_ws/src/smb_description/urdf/smb_standalone.urdf.xacro', 'wheel_joint_type:=continuous', 'simulation:=true', 'robot_namespace:=', 'lidar:=true', 'description_name_xacro:=robot_description', 'publish_tf:=false']] returned with code [2]. 

Param xml is <param name="$(arg description_name)" command="$(find xacro)/xacro     $(arg description_file)     wheel_joint_type:=$(arg wheel_joint_type)     simulation:=$(arg simulation)     robot_namespace:=$(arg robot_namespace)     lidar:=$(arg lidar)     description_name_xacro:=$(arg description_name)     publish_tf:=$(arg publish_tf)"/>

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3/dist-packages/roslaunch/__init__.py", line 347, in main
    p.start()
  File "/opt/ros/noetic/lib/python3/dist-packages/roslaunch/parent.py", line 305, in start
    self._start_infrastructure()
  File "/opt/ros/noetic/lib/python3/dist-packages/roslaunch/parent.py", line 254, in _start_infrastructure
    self._load_config()
  File "/opt/ros/noetic/lib/python3/dist-packages/roslaunch/parent.py", line 156, in _load_config
    self.config = roslaunch.config.load_config_default(self.roslaunch_files, self.port,
  File "/opt/ros/noetic/lib/python3/dist-packages/roslaunch/config.py", line 463, in load_config_default
    raise RLException(e)
roslaunch.core.RLException: while processing /home/theanvin/Workspaces/smb_ws/src/smb_control/launch/control.launch:
while processing /home/theanvin/Workspaces/smb_ws/src/smb_description/launch/load.launch:
Invalid <param> tag: Cannot load command parameter [robot_description]: command [['/opt/ros/noetic/lib/xacro/xacro', '/home/theanvin/Workspaces/smb_ws/src/smb_description/urdf/smb_standalone.urdf.xacro', 'wheel_joint_type:=continuous', 'simulation:=true', 'robot_namespace:=', 'lidar:=true', 'description_name_xacro:=robot_description', 'publish_tf:=false']] returned with code [2]. 

Param xml is <param name="$(arg description_name)" command="$(find xacro)/xacro     $(arg description_file)     wheel_joint_type:=$(arg wheel_joint_type)     simulation:=$(arg simulation)     robot_namespace:=$(arg robot_namespace)     lidar:=$(arg lidar)     description_name_xacro:=$(arg description_name)     publish_tf:=$(arg publish_tf)"/>

[rospy.core][INFO] 2024-03-05 04:55:17,593: signal_shutdown [atexit]
